client/asset/dcr/spv.go

```diff
@@ -939,7 +938,7 @@ var (
 // logRotator initializes a rotating file logger.
 func logRotator(netDir string) (*rotator.Rotator, error) {
 	const maxLogRolls = 8
-	logDir := filepath.Join(netDir, logDirName)
+	logDir := filepath.Join(filepath.Dir(netDir), logDirName)
```

Comment:
Should be just telling `logRotator` the right parent dir to use, not have `logRotator` strip a path.  Here's the right place:

https://github.com/decred/dcrdex/blob/441eeb3df87ada71afe0565d624a581793403a76/client/asset/dcr/spv.go#L141-L142

Don't pass that `dir` and expect `logRotator` to strip off `"spv"`.  Instead have an intermediate `netDir := filepath.Join(dataDir, chainParams.Name)` That you give to `logRotator`